### PR TITLE
feat: update landing page footer

### PR DESCRIPTION
**Pull request by karrui**
## Problem

This PR updates the design of the landing page footer.

Related to https://github.com/opengovsg/FormSG/issues/369

## Solution

**Features**:

- New footer on the landing page

## After Screenshots

**XL**
Screenshot 2020-11-04 at 9 54 25 AM

**L**
Screenshot 2020-11-04 at 9 54 31 AM

**Tablet**
Screenshot 2020-11-04 at 9 54 35 AM

**Mobile**
Screenshot 2020-11-04 at 9 54 47 AM

## Tests

Frontend change has been tested on Chrome and IE11 in all responsive resolutions.

### CODE REVIEW

**Comment by pearlyong** [comment]
i think @syan-syan mentioned to change FormSG to Form? otherwise looks good.

**Comment by karrui** [comment]
> i think @syan-syan mentioned to change FormSG to Form? otherwise looks good.

Done.

File: src/public/modules/core/componentViews/footer.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        translate="FOOTER.PRIVACY.TEXT"
+      ></a>
+      <a
+        class="footer__links--link"
+        ui-sref="terms"
+        target="_blank"
+        translate="FOOTER.TOS.TEXT"
+      ></a>
+      <a
+        class="footer__links--link footer__links--link--last"
```
Comment (mantariksh) [inline src/public/modules/core/componentViews/footer.html:27]: for the record I don't think this is what BEM is supposed to look like lol but probably not worth fixing

File: src/public/translations/en-SG/landing.json [hunk 2]
```diff
@@ -110,7 +110,7 @@
     }
   },
   "FOOTER": {
-    "APP_NAME": "FormSG",
+    "APP_NAME": "Form",
```
Comment (mantariksh) [inline src/public/translations/en-SG/landing.json:113]: are we really doing this here? even in the sg version?

**Review by mantariksh — APPROVED** [review]
did I even look at the code? or did I just look at the screenshots and approve? the world will never know

File: src/public/translations/en-SG/landing.json (hunk 2 above)
Comment (mantariksh) [reply src/public/translations/en-SG/landing.json:113]: it's not even gonna look like a brand, it's just gonna look like the word "Form" lol

Comment (karrui) [reply src/public/translations/en-SG/landing.json:113]: Doing this according to the design haha 🤷

File: src/public/modules/core/componentViews/footer.html (hunk 1 above)
Comment (karrui) [reply src/public/modules/core/componentViews/footer.html:27]: no h8, no h8

File: src/public/translations/en-SG/landing.json (hunk 2 above)
Comment (mantariksh) [reply src/public/translations/en-SG/landing.json:113]: am I the only one who thinks the word Form in "**Form** _Build government forms in minutes_" just looks like the word Form and not a brand

Comment (mantariksh) [reply src/public/translations/en-SG/landing.json:113]: @pearlyong thoughts?

Comment (syan-syan) [reply src/public/translations/en-SG/landing.json:113]: can we see it in staging to have a better sense? 

Go has 'Go.gov.sg' in both the website title and in the footer; the other products have mostly identifiable brand names.

**Pull request merged**